elo_boat.py

```diff
@@ -728,9 +747,10 @@ def disp_elo(player_elo, convergence):
 # ==============================================================================archi
 # ==============================================================================archi
 
-async def not_admin(ctx):
+async def not_admin(ctx, display_message=True):
     if ctx.message.author.roles[-1] < _guild.get_role(admin_role_id) and ctx.message.author.id != 230018748491235339:
-        await ctx.channel.send(NO_POWER_MSG)
+        if display_message:
+            await ctx.channel.send(NO_POWER_MSG)

```

Comment:
In what world you want not to advise ppl that the problem is not that they're inputting the command wrong, but that they doesn't have the required permissions ?